### DISCUSSION TITLE
Need help to downgrade preinstalled netflix app to this mod

### DISCUSSION THREAD

**Discussion by Archonide** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#21] [General]
Worked fine on my Skyworth Box (operator-tier) before updating the device to Android 10. The update came with preinstalled netflix app at system level. I can downgrade the official netflix app through adb command, down to the lowest version that supports API 23, but cannot downgrade to this mod due signature mismatched. The device has Widevine L1, but is not netflix-certified due to licensing issues with the TV operator. Any workaround on how to install this mod?

**Comment by aidanmacgregor** [comment]
Maybe changing the package name (from com.netflix.xxx. Along those lines) to something different, then it wont try to install over your official Netflix, this may break the app but worth a shot :)

**Reply by piotrekcrash** [reply]
@Archonide did you solve somehow?
